test/group.js

```diff
@@ -114,7 +114,7 @@ describe('group methods (no default rights)', () => {
 
   test('should get list of groups user is member of, including default groups', async function () {
     let groups = await couch.getUserGroups('a@a.com');
-    groups.sort((a, b) => b.name < a.name);
+    groups.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
Unrelated to this PR but the other way failed on my computer with node canary (probably a temporary V8 bug)